### PR TITLE
add variables and first profile

### PR DESCRIPTION
**Pull request by mizzrawr**
pls check my super mega awesome code

### CODE REVIEW

**Review by ghassanaldarwish — CHANGES_REQUESTED** [review]
Please create a folder with your name holds JavaScript file

File: index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// 1. console log a line of text
+console.log("This is the first exercise");
+
```
Comment (ciamac-da) [inline index.js]: The content of console.log should be "this is an exercise in console logging" :))

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hola Cindy:) You forgot to make a folder! You have to make a folder and put your index.js inside of this folder! And there's also a tiny mistake about the first  task. The content of console.log should be "this is an exercise in console logging" :))

File: index.js (hunk 1 above)
Comment (mizzrawr) [inline index.js]: ``console.log("This is an exercise in console logging");

**Pull request merged**